storage-stack-automation/storage_stack.yaml

```diff
@@ -0,0 +1,291 @@
+          def add_storage(bucket_name, ext_id, account_id, kms_arn):
+              # deploy storage stack
+              ExternalID = {"ParameterKey": "ExternalID", "ParameterValue": ext_id}
+              S3BucketToScan = {"ParameterKey": "S3BucketToScan", "ParameterValue": bucket_name}
+              Trigger_with_event = {
+                  "ParameterKey": "TriggerWithObjectCreatedEvent",
+                  "ParameterValue": "true",
+              }
+              scanner_queue_url = {"ParameterKey": "ScannerSQSURL", "ParameterValue": queue_name}
+              scanner_aws_account = {
+                  "ParameterKey": "ScannerAWSAccount",
+                  "ParameterValue": account_id,
+              }
+              S3_Encryption = {"ParameterKey": "KMSKeyARNForBucketSSE", "ParameterValue": kms_arn}
+              cft_client = boto3.client("cloudformation")
+              logger.info("Creating stack ..")
+              cft_client.create_stack(
+                  StackName="C1-FSS-Storage-" + bucket_name,
+                  TemplateURL="https://file-storage-security.s3.amazonaws.com/latest/templates/FSS-Storage-Stack.template",
+                  Parameters=[
+                      ExternalID,
+                      S3BucketToScan,
+                      scanner_queue_url,
+                      Trigger_with_event,
+                      scanner_aws_account,
+                      S3_Encryption,
+                  ],
+                  Capabilities=["CAPABILITY_IAM"],
+              )
+              cft_waiter = cft_client.get_waiter("stack_create_complete")
+              cft_waiter.wait(StackName="C1-FSS-Storage-" + bucket_name)
+              res = cft_client.describe_stacks(StackName="C1-FSS-Storage-" + bucket_name)
+              storage_stack = res["Stacks"][0]["Outputs"][2]["OutputValue"]
+              logger.info("FSS StorageRole Arn: " + storage_stack)
+              logger.info(storage_stack)
+              # add to c1
+              payload = {
+                  "type": "storage",
+                  "scannerStack": stack_id,
+                  "provider": "aws",
+                  "details": {"managementRole": storage_stack},
+              }
+              encoded_msg = json.dumps(payload).encode("utf-8")
+              resp = http.request(
+                  "POST",
+                  post_url,
+                  headers={
+                      "Content-Type": "application/json",
+                      "api-secret-key": fss_key,
+                      "Api-Version": "v1",
+                  },
+                  body=encoded_msg,
+              )
+              transform = json.loads(resp.data.decode("utf-8"))
+              logger.info(transform)
+              logger.info("Storage Stack has deployed")
+      MemorySize: 128
+      Environment:
+        Variables:
+          C1_API: !Ref C1API
+          SQS_Name: !Ref SQSURL
+          STACK_ID: !Ref StackID
+  cloudwatchevent:
+    Type: AWS::Events::Rule
```

Comment:
It's actually being renamed from CloudWatch event to EventBridge event.

Reply:
renamed to eventbridgeevent